src/pages/index.page.tsx

```diff
@@ -16,66 +17,98 @@ import { ACTIONS, StoreContext } from 'store';
 const CURRENT_DATE_FORMAT = 'cccc, LLLL d';
 
 const SurveyListPlaceholder = () => (
-  <section className="flex flex-col">
+  <div className="mt-24 text-center">
     <Image width={64} height={64} src="/icon/smiley.svg" alt="smiley icon" />
     <h2 className="mt-8 text-3xl font-extrabold text-center">
       You&#39;ve completed all the surveys.
       <br /> Take a moment.
     </h2>
-  </section>
+  </div>
 );
 
 const Home: NextPage = () => {
   useSession();
-  const [surveyFetching, setSurveyFetching] = useState(true);
   const { setBgUrl } = useContext(BackgroundContext);
-  const { store, dispatchAction } = useContext(StoreContext);
+  const {
+    store: {
+      surveys: {
+        data: surveys,
+        pageInfo: { currentPage, totalPages },
+      },
+      userProfile,
+    },
+    dispatchAction,
+  } = useContext(StoreContext);
 
-  const currentDate = new Date();
+  const [surveyFetching, setSurveyFetching] = useState(true);
 
   useEffect(() => {
-    setBgUrl(null);
+    setBgUrl(surveys[0]?.coverImageUrl || null);
   }, [setBgUrl]);
 
   useEffect(() => {
-    listSurveys().then((surveys) => {
-      dispatchAction({ type: ACTIONS.SET_SURVEYS, value: surveys });
+    listSurveys().then(({ surveys, meta }) => {
+      dispatchAction({
+        type: ACTIONS.SET_SURVEYS,
+        value: {
+          data: surveys,
+          pageInfo: { currentPage: 1, totalPages: meta.pages },
+        },
+      });
```

Comment:
I think I'm coming a little bit late with that, so feel free to ignore it for this project. 
I'm still sharing it as IMO it is interesting 😇 

Should we use the [Flux Standard Action Convention](https://redux.js.org/style-guide/#write-actions-using-the-flux-standard-action-convention), as recommended by Redux? 🤔 

This would change to something like: 
```suggestion
      dispatchAction({
        type: ACTIONS.SET_SURVEYS,
        payload: surveys,
        meta: { currentPage: 1, totalPages: meta.pages },
      });
```
I like it as it makes a clear distinction between the data and the meta-data (and errors if any). This can be important when considering the payload is immutable and should be a serializable data structure, while meta data and errors could be any object needed by the action to its process (aka it won't be stored in the store).

Feel free to keep the current way (note also that Redux has it a priority C only, so it is absolutely **not** critical ;-) )

![image](https://user-images.githubusercontent.com/77609814/174015253-f7b9f79f-1b66-40c2-aaff-41b78283bf8b.png)

Reply:
Definitely an interesting guideline! I may update this after completing the first phase of IC as it needs to update in several places.